### DISCUSSION TITLE
Quick start Error

### DISCUSSION THREAD

**Discussion by harasandayo** [sse-secure-systems/connaisseur#525] [Q&A]
Hi all,

I have challenged Quick start 'https://sse-secure-systems.github.io/connaisseur/v2.2.0/'
And when I did the following, I got a title error 
   $ kubectl run hello-world --image=docker.io/hello-world

**Comment by xopham** [comment]
hi @harasandayo :wave:
which flavor/version of kubernetes do you use? is there general internet connectivity provided or restrictions by a firewall in place?

**Reply by harasandayo** [reply]
Hi　@xopham thanks for replying.
I use k3s/v1.22.6+k3s1.
I think restrictions by a firewall.

**Reply by xopham** [reply]
@harasandayo we are testing with k3s against all kubernetes versions. I would presume that it must be either some kind of firewall or network policy or a DNS problem or general lack of internet connectivity.
The fact that it yields "name does not resolve" already indicates that it fails at the level of DNS already before even attempting a connection.
Please let me know if you were able to get it resolved :pray:
